app/presenters/export/csv/project_presenter.rb

```diff
@@ -182,4 +182,32 @@ def trust_surplus_deficit
     return I18n.t("export.csv.project.values.not_applicable") if @project.tasks_data.check_and_confirm_financial_information_not_applicable?
     @project.tasks_data.check_and_confirm_financial_information_trust_surplus_deficit
   end
+
+  def diocese_name
+    @project.establishment.diocese_name
+  end
+
+  def diocese_contact
+    @project.contacts.where(category: "diocese")
+  end
+
+  def advisory_board_conditions
+    @project.advisory_board_conditions
+  end
+
+  def completed_grant_payment_certificate_received
+    @project.receive_grant_payment_certificate_date_received if @project.tasks_data.receive_grant_payment_certificate_date_received.present?
+  end
+
+  def solicitor_contact
+    @project.contacts.where(category: "solicitor")
+  end
+
+  def project_created_by
+    @project.regional_delivery_officer_id
+  end
+
+  def team_managing_the_project
+    @project.team
```

Comment:
This will return a database-friendly string, e.g. `west_midlands`. Can you return the associated I18n string, e.g. `teams.west_midlands` which will resolve to `West Midlands`